proctor-common/src/main/java/com/indeed/proctor/common/ProctorResult.java

```diff
@@ -43,6 +44,8 @@ public class ProctorResult {
 
     private final Map<String, Object> inputContext;
 
+    private final HashSet<String> hasLoggedTests;
```

Comment:
NIT: can rename to just loggedTests

Reply:
IMHO, "loggedTestVersions" is a bit confusing, as "version" means one test different versions